cou/steps/analyze.py

```diff
@@ -136,18 +136,24 @@ def is_data_plane(app: OpenStackApplication) -> bool:
         return control_plane, data_plane
 
     @classmethod
-    async def create(cls, model: juju_utils.Model) -> Analysis:
+    async def create(
+        cls, model: juju_utils.Model, skip_apps: Optional[list[str]] = None
```

Comment:
```suggestion
        cls, model: juju_utils.Model, skip_apps: list[str]
```

Reply:
I am aware of this, but this is also why I don't want to make this change: it breaks compatibility of unit tests since it's not optional

Reply:
>  it breaks compatibility of unit tests since it's not optional

This should never be a reason to not improve application code.

Reply:
but it did not improve the code's logic: that `skip_apps` is supposed to be optional not a requirement

Reply:
It made the typing more accurate and removed redundant logic (the `or []` part). :)